db/Schema.sql

```diff
@@ -32,12 +32,11 @@ CREATE TABLE products (
 DROP TABLE IF EXISTS cart CASCADE;
 
 CREATE TABLE cart (
-  cart_id SERIAL NOT NULL,
   "user_id" INTEGER NOT NULL,
   product_id INTEGER NOT NULL,
   cart_quantity INTEGER NOT NULL,
 
-  PRIMARY KEY (cart_id)
+  PRIMARY KEY ("user_id", product_id)
```

Comment:
I think there can only be one primary key on a given table- correct me if I'm wrong. `product_id` sounds like it would be a foreign key in this  table.

Reply:
@jessicaawhy 

I re added the cart_id. 

Currently, the cart_id is functionally just a unique combination of each user_id and product_id. Postgres allows you to make primary keys out of combinations of other columns, which creates a hidden index for that unique combination. I needed the unique combination to resolve conflicts in the putCart API call. I've now added back the cart_id primary key and created a separate unique constraint to the user_id/product_id columns.